core/base/mergeTreeDistanceMatrix/MergeTreeDistanceMatrix.h

```diff
@@ -29,6 +31,11 @@ namespace ttk {
    */
   class MergeTreeDistanceMatrix : virtual public Debug,
                                   virtual public MergeTreeBase {
+  protected:
+    int baseModule = 0;
```

Comment:
can you rename into `baseModule_` ?(the `_` post-fix is used throughout the base code layer to denote member variables)